launch/vive.launch

```diff
@@ -1,17 +1,34 @@
-  <env name="STEAM" value="$(env HOME)/.local/share/Steam"/>
-  <env name="STEAMVR" value="$(env HOME)/.local/share/Steam/steamapps/common/SteamVR"/>
+  <arg name="OPENVR"  value="$(env HOME)/libraries/openvr"/>
+  <arg name="STEAM"   value="$(env HOME)/.local/share/Steam"/>
+  <arg name="STEAMVR" value="$(arg STEAM)/steamapps/common/SteamVR"/>
+
+  <env name="OPENVR"  value="$(arg OPENVR)"/>
+  <env name="STEAM"   value="$(arg STEAM)"/>
+  <env name="STEAMVR" value="$(arg STEAMVR)"/>
+	    
   <!-- Everything! -->
 <!--
   <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):$(env HOME)/libraries/openvr/lib/linux32:$(env HOME)/libraries/openvr/lib/linux64/:\$(env HOME)/.local/share/Steam/ubuntu12_32/steam-runtime/i386/lib/i386-linux-gnu:$(env HOME)/.local/share/Steam/ubuntu12_32/steam-runtime/amd64/lib/x86_64-linux-gnu:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/bin/linux32:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/bin/linux64:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/linux32:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/linux64:"/>
 -->
   <!-- For a 64bit system -->
-  <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):$(env HOME)/libraries/openvr/lib/linux64:$(env HOME)/.local/share/Steam/ubuntu12_32/steam-runtime/amd64/lib/x86_64-linux-gnu:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/bin/linux64:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/linux64"/>
+  <!-- <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):$(env HOME)/libraries/openvr/lib/linux64:$(env HOME)/.local/share/Steam/ubuntu12_32/steam-runtime/amd64/lib/x86_64-linux-gnu:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/bin/linux64:$(env HOME)/.local/share/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/linux64"/> -->
+  <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):/usr/lib/:/usr/lib32/:$(arg OPENVR)/lib/linux32/:$(arg OPENVR)/lib/linux64/:$(arg STEAM)/ubuntu12_32/steam-runtime/i386/usr/lib/i386-linux-gnu/:$(arg STEAM)/ubuntu12_32/steam-runtime/amd64/usr/lib/x86_64-linux-gnu/:$(arg STEAM)/ubuntu12_32/steam-runtime/amd64/lib/x86_64-linux-gnu/:$(arg STEAM)/ubuntu12_32/steam-runtime/i386/lib/i386-linux-gnu/:$(arg STEAMVR)/bin/linux32/:$(arg STEAMVR)/bin/linux64/:$(arg STEAMVR)/drivers/lighthouse/bin/linux32/:$(arg STEAMVR)/drivers/lighthouse/bin/linux64/"/>
+    
 
   <rosparam param="/vive/world_offset">[0, 0, 2]</rosparam>
   <rosparam param="/vive/world_yaw">0.0</rosparam>
   
+  <!-- <remap from="image_left" to="/multisense_local/left/image_rect_color"/> -->
+  <!-- <remap from="image_right" to="/multisense_local/right/image_rect"/> -->
+  <!-- <remap from="camera_info_left" to="/multisense_local/left/camera_info"/> -->
+  <!-- <remap from="camera_info_right" to="/multisense_local/right/camera_info"/> -->
+  
+  <remap from="image_left" to="/usb_cam/image_raw"/>
```

Comment:
I see.
But even between usb_cam and multisense, the image topic field names are different (image_raw and image_rect_color)
Is it really useful that introducing CAMERA arg ?